view/main/index.ts

```diff
@@ -28,26 +25,6 @@ expand.addEventListener('click', () => {
   document.body.classList.toggle('is-main-collapsed')
 })
 
-const addHoverLogo = (): void => {
-  emLogo.classList.add('is-hover')
-}
-const removeHoverLogo = (): void => {
-  emLogo.classList.remove('is-hover')
-}
-
-emText.addEventListener('mouseenter', addHoverLogo)
-emText.addEventListener('mouseleave', removeHoverLogo)
-emText.addEventListener('focus', addHoverLogo)
-emText.addEventListener('blur', removeHoverLogo)
```

Comment:
As I see, `:has` was already used. So it is make sense to synchronize the hover of these links with CSS only.
Besides, this is obviously not core functionality, so there is no point even think about polyfill that feature.

Reply:
Good idea